x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/analytics_table.scss

```diff
@@ -0,0 +1,5 @@
+.entSearch__table--noBackground {
+  .euiTable {
+    background-color: transparent;
+  }
+}
```

Comment:
A bit of a workaround right now to use the tables on the `subdued` panels for amsterdam. Until this is officially supported in EUI, this should suffice.